wormhole-connect/src/views/Bridge/Bridge.tsx

```diff
@@ -231,21 +246,29 @@ function Bridge() {
         }
       }
     };
+
     computeDestTokens();
+
     return () => {
       canceled = true;
     };
     // IMPORTANT: do not include destToken in dependency array
   }, [route, token, fromChain, toChain, dispatch]);
 
   useEffect(() => {
+    if (!route || !amount || !token || !destToken || !fromChain || !toChain) {
```

Comment:
@artursapek I assume if one of these are missing, we don't need to calculate the receive amount, right?